packages/@aws-cdk/aws-certificatemanager/lib/certificate.ts

```diff
@@ -87,6 +92,16 @@ export interface CertificateProps {
    * @default true
    */
   readonly transparencyLoggingEnabled?: boolean;
+
+
+  /**
+   * The Certifcate name.
+   *
+   * Since the Certifcate resource doesn't support providing a physical name, the value provided here will be recorded in the `Name` tag
+   *
+   * @default this.node.path
```

Comment:
```suggestion
   * @default - the full, absolute path of this construct
```

I think something like this might make more sense to the user.